sdk_lib/sdk_container_common.sh

```diff
@@ -47,6 +47,10 @@ function yell() {
 # Guess the SDK version from the current git commit.
 #
 function get_git_version() {
+    if [[ -f '.git_version' ]]; then
+        cat '.git_version'
```

Comment:
If that is useful in general, how about an env var instead?

Reply:
Not sure about usefulness - it's just my setup that does not work well with the script (I use git worktree functionality extensively). Still, it's a hack and my gut feeling is that this should be stored as a file somewhere in the scripts repo. Environment variable can be a bit finicky when involved with a bunch of sudos and dockers along the way.

Reply:
It's already a part of boilerplate stuff that is already in main.